src/lib/components/player/rank-chart.svelte

```diff
@@ -11,6 +11,15 @@
    });
 
    history.push(player.rank);
+   
+   // If this player stopped playing for a while, but has picked up again, then remove the 
+   // #999999 ranks at the beginning to make the chart more useful
+   const inactiveRankPlaceholder = 999999;
+   const firstActiveRankIndex = history.findIndex(h => h !== inactiveRankPlaceholder);
+   if (firstActiveRankIndex >= 0) {
+      history.splice(0, firstActiveRankIndex);
```

Comment:
~~Isn't that also hiding the first day of activity?~~ Just double checked the docs, all good.